// Copyright (c) YugaByte, Inc.

package yb.docdb;

option java_package = "org.yb.docdb";

message KeyValuePairPB {
  optional bytes key = 1;
  optional bytes value = 2;
}

// A set of key/value pairs to be written into RocksDB.
message KeyValueWriteBatchPB {
  repeated KeyValuePairPB kv_pairs = 1;
}
